MediaBrowser.Providers/TV/DummySeasonProvider.cs

```diff
@@ -127,17 +127,24 @@ private async Task<bool> AddDummySeasonFolders(Series series, CancellationToken
             bool isVirtualItem,
             CancellationToken cancellationToken)
         {
-            var seasonName = seasonNumber == 0 ?
-                _libraryManager.GetLibraryOptions(series).SeasonZeroDisplayName :
-                (seasonNumber.HasValue ? string.Format(_localization.GetLocalizedString("NameSeasonNumber"), seasonNumber.Value.ToString(_usCulture)) : _localization.GetLocalizedString("NameSeasonUnknown"));
+            var seasonName = seasonNumber == 0
+                ? _libraryManager.GetLibraryOptions(series).SeasonZeroDisplayName
+                : (seasonNumber.HasValue
+                    ? string.Format(
+                        CultureInfo.InvariantCulture,
+                        _localization.GetLocalizedString("NameSeasonNumber"),
+                        seasonNumber.Value)
+                    : _localization.GetLocalizedString("NameSeasonUnknown"));
```

Comment:
this looks like uglified JS we're battling with... :) can this huge ternary hell be replaced with nice chain of `if`-s?